.github/workflows/release-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         platform: ["x86_64", "aarch64"]
-        os: ["ubuntu:20.04", "ubuntu:22.04", "debian:11-slim", "fedora:37"]
```

Comment:
This distribution is used for testing Python modules, not for building them or MeshLib itself.